webpack/assets/javascripts/react_app/components/Layout/LayoutReducer.js

```diff
@@ -37,10 +37,10 @@ export default (state = initialState, action) => {
       return state.set('isLoading', false);
 
     case LAYOUT_CHANGE_ORG:
-      return state.set('currentOrganization', payload.organization);
+      return state.set('isLoading', true);
```

Comment:
As mentioned in https://github.com/theforeman/foreman/pull/8255#issuecomment-796599231 IMHO we don't need fancy UI loader here, when this is indicated by browser loading page state.

Reply:
I tend to agree, I see it confusing having 2 loading indicators (browser and in the UI).
IMO we should use them to indicate that the UI is working on sending-receiving data.

Reply:
I don't think it's too bad to show the loading indicator, we have pages where it takes a few seconds to do a full-refresh,
and until that happens, the data which is visible is from the prior org/loc...
ideally, I would show some loading skeleton for each page, but that should be on another PR,
here's an example of how it looks now:

![Untitled_ Mar 14, 2021 5_01 PM](https://user-images.githubusercontent.com/26363699/111073626-1be1e400-84e8-11eb-940a-dc574c345cf6.gif)

Reply:
I believe there shouldn't be redux reducer and action for this even if we want to do so :)
Redux is for maintaining the data. I disagree with the point of "we see data from the previous taxonomy", we do, because the page didn't reload. The UI stays correctly on the old taxonomy, so it is still showing the old data and is in reload page state.